### DISCUSSION TITLE
FAILED

### DISCUSSION THREAD

**Discussion by serkanaklan** [saran-surya/email_auth_node#60] [Q&A]
Hi I execute commands, but I don't get results. I ran this code in terminal "email-auth-node --version" error "The term 'email-auth-node' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again. At line:1 char:1 + email-auth-node --version + ~~~~~~~~~~~~~~~ + CategoryInfo : ObjectNotFound: (email-auth-node:String)[], CommandNotFoundException + FullyQualifiedErrorId : CommandNotFoundException" When I try to use the package with flutter, is this a factor that does not work?

**Comment by saran-surya** [comment]
Did you perform ```npm i && npm link``` from the root. And you can alternatively try ```npx email-auth-node --version```
